.github/workflows/update-core.yaml

```diff
@@ -0,0 +1,43 @@
+name: Update Core to Latest Commit
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # every day at midnight
+
+jobs:
+  update-pomerium-core:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491
+        with:
+          go-version: 1.22.x
+      - name: Update Core
+        run: |
+          go get -u github.com/pomerium/pomerium@main
+          go mod tidy
+      - name: Check for changes
+        id: git-diff
+        run: |
+          git config --global user.email "apparitor@users.noreply.github.com"
+          git config --global user.name "GitHub Actions"
+          git add go.mod go.sum
+          git diff --cached --exit-code || echo "::set-output name=changed::true"
```

Comment:
The `::set-output` syntax is deprecated (although I don't know that GitHub is ever going to go through with removing it). Still, would you mind updating this to the [`$GITHUB_OUTPUT`](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) mechanism instead?